src/WARDuino/WARDuino.cpp

```diff
@@ -823,37 +823,51 @@ Module *WARDuino::load_module(uint8_t *bytes, uint32_t byte_count,
     }
 
     find_blocks(m);
+    debug("findblocks finished\n");
 
     if (m->start_function != UNDEF) {
+        debug("has startfun \n");
         uint32_t fidx = m->start_function;
+        debug("1 \n");
```

Comment:
What are these messages for?

Reply:
These were added in commit 23ae473ef1e6f7ac45998d7d0650e525406fff36. I suspect these were never meant to be commited. I will remove them.

Reply:
Removed in 1fe5979